interfaces/console/pages/site_planning.tsx

```diff
@@ -168,13 +168,13 @@ const Page = () => {
     variables: {
       userId: _commonData.userId,
     },
-    onCompleted: (data) => {
+    onCompleted: (data: any) => {
```

Comment:
I think we have a type of this data https://github.com/ukama/ukama/blob/bf8d25017e2b9f8e5ac0f937589c2563f9ff44af/interfaces/console/generated/index.ts#L749C13-L749C18

Reply:
addressed